<!DOCTYPE html>
<html >
<head>
  <!-- Site made with Mobirise Website Builder v4.6.3, https://mobirise.com -->
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Mobirise v4.6.3, mobirise.com">
  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
  <link rel="shortcut icon" href="assets/images/screenshot.png" type="image/x-icon">
  <meta name="description" content="Landing page/ general info">
  <title>Home</title>
  <link rel="stylesheet" href="assets/web/assets/mobirise-icons/mobirise-icons.css">
  <link rel="stylesheet" href="assets/tether/tether.min.css">
  <link rel="stylesheet" href="assets/bootstrap/css/bootstrap.min.css">
  <link rel="stylesheet" href="assets/bootstrap/css/bootstrap-grid.min.css">
  <link rel="stylesheet" href="assets/bootstrap/css/bootstrap-reboot.min.css">
  <link rel="stylesheet" href="assets/dropdown/css/style.css">
  <link rel="stylesheet" href="assets/theme/css/style.css">
  <link rel="stylesheet" href="assets/mobirise/css/mbr-additional.css" type="text/css">
  
  
  
</head>
<body>
  <section class="menu cid-qHM916KSbE" once="menu" id="menu2-1">

    

    <nav class="navbar navbar-expand beta-menu navbar-dropdown align-items-center navbar-fixed-top navbar-toggleable-sm">
        <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
            <div class="hamburger">
                <span></span>
                <span></span>
                <span></span>
                <span></span>
            </div>
        </button>
        <div class="menu-logo">
            <div class="navbar-brand">
                
                <span class="navbar-caption-wrap"><a class="navbar-caption text-white display-4" href="index.html">
                        Iddo Friedberg lab</a></span>
            </div>
        </div>
        <div class="collapse navbar-collapse" id="navbarSupportedContent">
            <ul class="navbar-nav nav-dropdown nav-right" data-app-modern-menu="true"><li class="nav-item"><a class="nav-link link text-white display-4" href="software.html">
                        Software</a></li><li class="nav-item"><a class="nav-link link text-white display-4" href="publications.html">
                        Publications</a></li><li class="nav-item"><a class="nav-link link text-white display-4" href="people.html">
                        People</a></li><li class="nav-item"><a class="nav-link link text-white display-4" href="positions.html">
                        Positions</a></li><li class="nav-item"><a class="nav-link link text-white display-4" href="contacts.html">
                        Contact Us</a></li></ul>
            
        </div>
    </nav>
</section>

<section class="engine"><a href="https://mobirise.ws/l">free html site builder</a></section><section class="header12 cid-qJt2ByxrZm" id="header12-2h">

    

    

    <div class="container  ">
            <div class="media-container">
                <div class="col-md-12 align-center">
                    <h1 class="mbr-section-title pb-3 mbr-white mbr-bold mbr-fonts-style display-1">Iddo Friedberg Lab</h1>
                    <p class="mbr-text pb-3 mbr-white mbr-fonts-style display-5">
                        Solving biological problems with computation. Special love for bacteria. Protein function prediction, bacterial evolution, metagenomics. 
                    </p>
                    <div class="mbr-section-btn align-center py-2"><a class="btn btn-md btn-secondary display-4" href="positions.html">Open Positions</a></div>

                    <div class="icons-media-container mbr-white">
                        <div class="card col-12 col-md-6 col-lg-3">
                            <div class="icon-block">
                            <a href="software.html">
                                <span class="mbr-iconfont mbri-laptop"></span>
                            </a>
                            </div>
                            <h5 class="mbr-fonts-style display-5">
                                Software</h5>
                        </div>

                        <div class="card col-12 col-md-6 col-lg-3">
                            <div class="icon-block">
                                <a href="publications.html">
                                    <span class="mbr-iconfont mbri-file"></span>
                                </a>
                            </div>
                            <h5 class="mbr-fonts-style display-5">
                                Publications</h5>
                        </div>

                        <div class="card col-12 col-md-6 col-lg-3">
                            <div class="icon-block">
                                <a href="people.html">
                                    <span class="mbr-iconfont mbri-users"></span>
                                </a>
                            </div>
                            <h5 class="mbr-fonts-style display-5">
                                People</h5>
                        </div>

                        <div class="card col-12 col-md-6 col-lg-3">
                            <div class="icon-block">
                                <a href="contacts.html">
                                    <span class="mbr-iconfont mbri-letter"></span>
                                </a>
                            </div>
                            <h5 class="mbr-fonts-style display-5">
                                Contact Us</h5>
                        </div>
                    </div>
                </div>
            </div>
    </div>

    
</section>

<section class="mbr-section content6 cid-qJt0lDCSxt" id="content6-2c">
    
     
    
    <div class="container">
        <div class="media-container-row">
            <div class="col-12 col-md-8">
                <div class="media-container-row">
                    <div class="mbr-figure" style="width: 100%;">
                      <a href="https://www.the-scientist.com/August2014/co-occurrence_full1.jpg"><img src="assets/images/1.jpg" alt="Mobirise" title=""></a>  
                    </div>
                    <div class="media-content">
                        <div class="mbr-section-text">
                            <p class="mbr-text mb-0 mbr-fonts-style display-7"><strong>Metagenomics</strong> is the study of genomic material extracted directly from the environment. New sequencing technologies have enabled the study of whole populations of genomes taken from microbial communities in the field, as opposed to single species clonal cultures in the lab. Metagenomics offers a way to study how genomes evolve to cope with the microbial biotic and abiotic environments. Our lab helped developed a method to study the correlation between the human gut microbiota and gut gene expression. We are applying this method towards studying infant gut development the effect of gut microbes on human health and wellness.
<br>
<br></p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>

<section class="mbr-section content7 cid-qJt0mG873o" id="content7-2d">
          
    

    <div class="container">
        <div class="media-container-row">
            <div class="col-12 col-md-8">
                <div class="media-container-row">
                    <div class="media-content">
                        <div class="mbr-section-text">
                            <p class="mbr-text align-right mb-0 mbr-fonts-style display-7"><strong>Bacterial Genome Evolution:</strong> <br>Gene blocks are a common occurrence in bacteria: these are genes which lie close together on the chromosome, and may participate in a common cellular or biochemical function. Operons are gene blocks whose member genes are co-transcribed. We have developed a new method to describe the evolution of operons and gene blocks in bacteria. We describe a small set of evolutionary events that can take place in gene block evolution, and count these events to create a new type of molecular clock that tells us <a href="https://arxiv.org/abs/1501.00302">how fast or how slow certain gene blocks may have evolved</a>. We hope to learn how new funcitons are acquired by ensembles of genes such as these.</p>
                        </div>
                    </div>
                    <div class="mbr-figure" style="width: 100%;">
                     <img src="assets/images/2-new.png" alt="Mobirise" title="">  
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>

<section class="mbr-section content6 cid-qJt0qmmcVj" id="content6-2e">
    
     
    
    <div class="container">
        <div class="media-container-row">
            <div class="col-12 col-md-8">
                <div class="media-container-row">
                    <div class="mbr-figure" style="width: 100%;">
                      <a href="https://www.nature.com/articles/nmeth.2340"><img src="assets/images/3-new.jpg" alt="Mobirise" title=""></a>  
                    </div>
                    <div class="media-content">
                        <div class="mbr-section-text">
                            <p class="mbr-text mb-0 mbr-fonts-style display-7"><strong>Function Prediction
</strong><br>Another interest of ours is the prediction of protein function. Genomics, proteomics and various other ``-omics'' inundate us with sequence and structure information, but the biological functions of those proteins in many cases still eludes us. Computational prediction of protein and gene function is a rapidly growing research field in bioinformatics [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16772267?dopt=AbstractPlus">1</a>]. I am the co-organizer of the automated computational protein function prediction meetings: <a href="http://biofunctionprediction.org/">AFP</a>. The AFP meetings bring together researchers to discuss various methods for protein function prediction. My personal interest in function prediction lies in predicting function from protein structure [<a href="https://www.sciencedirect.com/science/article/pii/S0969212607001049?via%3Dihub">2</a>]. We have recently started work on predicting gene function based on its genomic context in bacteria, using both genomic and <a href="http://www.nature.com/nrmicro/focus/metagenomics/index.html">metagenomic </a>data towards that end.</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>

<section class="mbr-section content7 cid-qJt1zBXQ8L" id="content7-2f">
          
    

    <div class="container">
        <div class="media-container-row">
            <div class="col-12 col-md-8">
                <div class="media-container-row">
                    <div class="media-content">
                        <div class="mbr-section-text">
                            <p class="mbr-text align-right mb-0 mbr-fonts-style display-7"><strong>Structural Signatures
<br></strong>
<br>We are interested in locating ``structural signatures'' that span different protein folds. My working hypothesis is that there are short local structural commonalities between proteins that otherwise share no obvious structure or function. Detecting these commonalities can help us understand protein evolution, folding, and design. [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16084393?dopt=Abstract">3</a>] , [<a href="https://academic.oup.com/nar/article/33/suppl_2/W249/2505444?ijkey=4rPNQLgzrhUIWIY&keytype=ref">4</a>]</p>
                        </div>
                    </div>
                    <div class="mbr-figure" style="width: 100%;">
                     <a href="http://www.cell.com/cms/attachment/577177/4301185/gr4_lrg.jpg"><img src="assets/images/4-new.jpg" alt="Mobirise" title=""></a>  
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>

<section class="mbr-section content6 cid-qJt1AeesIZ" id="content6-2g">
    
     
    
    <div class="container">
        <div class="media-container-row">
            <div class="col-12 col-md-8">
                <div class="media-container-row">
                    <div class="mbr-figure" style="width: 100%;">
                      <a href="https://academic.oup.com/view-large/figure/2075524/btl310f3.jpeg/Using%20an%20alignment%20of%20fragment%20strings%20for%20comparing%20protein%20structures"><img src="assets/images/5-new.jpeg" alt="Mobirise" title=""></a>  
                    </div>
                    <div class="media-content">
                        <div class="mbr-section-text">
                            <p class="mbr-text mb-0 mbr-fonts-style display-7"><strong>Different Representations of Protein Structures
<br></strong><br>The computational representation of a protein's 3D structure is a challenging problem because of varying and often conflicting considerations: at first sight it seems that as far as information is concerned, more is better, hence the drive to atomic level description. However, elaboration on the atomic level can be very ``noisy'' and be time and memory intensive. Therefore we often ask what is the minimal information we need to achieve a specific task, without going into the unnecessary detail of representing each and every atom. I am interested in different computational representations of protein structures suitable for different tasks. In one study we have shown that a 1D representation of protein structures can be used for fast database searching and alignments, and still preserve relevant structural information. [<a href="http://iddo-friedberg.net/Friedberg-158.pdf">5</a>].</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>


  <script src="assets/web/assets/jquery/jquery.min.js"></script>
  <script src="assets/popper/popper.min.js"></script>
  <script src="assets/tether/tether.min.js"></script>
  <script src="assets/bootstrap/js/bootstrap.min.js"></script>
  <script src="assets/smoothscroll/smooth-scroll.js"></script>
  <script src="assets/dropdown/js/script.min.js"></script>
  <script src="assets/touchswipe/jquery.touch-swipe.min.js"></script>
  <script src="assets/theme/js/script.js"></script>
  
  
</body>
</html>